### ISSUE TITLE
Slider should be more a11y

### ISSUE THREAD

**Issue by ipip2005** [OfficeDev/office-ui-fabric-react#665]
Slider should follow the best practice to support up and down arrow keys.
use up key to increase, down key to decrease the value.
Like windows controls (they both support all arrow keys):
![image](https://cloud.githubusercontent.com/assets/7711735/20996873/c47ab2b4-bd3b-11e6-935d-6182cddb084c.png)

**Comment by dzearing** [comment]
Resolved and will be in tonight's publish.
